packs/slack/actions/post_message.py

```diff
@@ -1,6 +1,11 @@
 import json
 import httplib
 
+try:
+    from six.moves.urllib.parse import urlencode
```

Comment:
`six` is only available in the sandbox in v0.12dev.

(Well, to be accurate, it's should also be available now because we use "--system-site-packages" workaround which we should eventually get rid of)